pyServer/manifests/windows/aks

```diff
@@ -19,7 +19,8 @@ copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Admin.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Operational.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Operational.evtx
 
-
+echo,### Calico Windows logs
+copy,/CalicoWindows/logs/*.log
```

Comment:
From the previous work, this will help collect logs under directory `c:\CalicoWindows\logs` which is the right path.